src/main/native/jgskit.mak

```diff
@@ -23,69 +23,35 @@ ifeq (${PLATFORM},arm-linux64)
   LDFLAGS+= -DLINUX
   IS64SYSTEM=64
   OSINCLUDEDIR=linux
-else ifeq (${PLATFORM},ppc-aix32)
-  PLAT=ap
-  CC=xlc
-  CFLAGS= -qcpluscmt -q32 -qpic -DAIX
-  LDFLAGS= -G -q32 -blibpath:${AIX_LIBPATH}
 else ifeq (${PLATFORM},ppc-aix64)
   PLAT=ap
   CC=xlc
   CFLAGS= -qcpluscmt -q64 -qpic -DAIX -qhalt=w
   LDFLAGS= -G -q64 -blibpath:${AIX_LIBPATH}
   IS64SYSTEM=64
   OSINCLUDEDIR=aix
-else ifeq (${PLATFORM},ppc-linux32)
-  PLAT=xp
-  CFLAGS+= -m32 -DLINUX
-  LDFLAGS+= -m32
-else ifeq (${PLATFORM},ppc-linux64)
-  PLAT=xp
-  CFLAGS+= -DLINUX
-  LDFLAGS+= -m64
-  IS64SYSTEM=64
```

Comment:
Would it make sense to remove the `IS64SYSTEM` everywhere, since they are all 64bit ones now?

Reply:
Yes agreed that it makes sense to remove `IS64SYSTEM` as this is no longer needed as everything is 64 bit.

Reply:
Updated with https://github.com/IBM/OpenJCEPlus/pull/348/commits/97153a9c2dcb73771278d1c69605df14dd31a0b4